### PR TITLE
Deprecating as Silverlight is no longer installable and has been deprecated for 10 years.

### PR DESCRIPTION
**Pull request by brydeno**

### CODE REVIEW

File: rules/do-you-offer-out-of-browser-support/rule.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 archivedreason:
```
Comment (SamBWagner) [inline rules/do-you-offer-out-of-browser-support/rule.md:3]:
```suggestion
archivedreason: Deprecating as Silverlight is no longer installable and has been deprecated for 10 years.
```

**Review by SamBWagner — COMMENTED** [review]
No. Fix plz

**Comment by tiagov8** [comment]
> No. Fix plz

Hey @SamBWagner Can you provide more info here?

@brydeno I wouldn't mention for how long Silverlight has been deprecated, since this information will also become deprecated :)

**Comment by SamBWagner** [comment]
> > No. Fix plz
> 
> Hey @SamBWagner Can you provide more info here?
> 
> @brydeno I wouldn't mention for how long Silverlight has been deprecated, since this information will also become deprecated :)

yes! 

The link in the PR was borked. We were standing right next to each other. Sorry @tiagov8 . I should give context regardless of the situation.

@brydeno 

Please be sure to move your change into the "archive-reason" section of the frontmatter. :) (You've already done it)

**Review by SamBWagner — APPROVED** [review]
🚢

**Pull request merged**